lumicks/pylake/calibration.py

```diff
@@ -310,12 +360,57 @@ def from_field(hdf5, force_channel, time_field="Stop time (ns)") -> "ForceCalibr
         items = []
         for calibration_item in hdf5["Calibration"].values():
             if force_channel in calibration_item:
-                attrs = calibration_item[force_channel].attrs
+                attrs = dict(calibration_item[force_channel].attrs)
                 if time_field in attrs.keys():
+                    # Copy the timestamp at which the calibration was applied into the item
+                    attrs["Timestamp (ns)"] = calibration_item.attrs.get("Timestamp (ns)")
                     items.append(ForceCalibrationItem(attrs))
 
         return ForceCalibration(time_field=time_field, items=items)
 
+    def print_summary(self, tablefmt):
+        return tabulate(
+            (
+                (
+                    idx,
+                    datetime.datetime.fromtimestamp(int(item.applied_at / 1e9)).strftime("%x %X")
+                    if item.applied_at
+                    else "-",
+                    item.kind,
+                    f"{item.stiffness:.2f}" if item.stiffness else "N/A",
+                    f"{item.force_sensitivity:.2f}" if item.force_sensitivity else "N/A",
+                    f"{item.displacement_sensitivity:.2f}" if item.force_sensitivity else "N/A",
+                    item.hydrodynamically_correct,
+                    item.distance_to_surface is not None,
+                    bool(
+                        self._slice_start
+                        and (item.start >= self._slice_start)
+                        and self._slice_stop
+                        and (item.stop <= self._slice_stop)
+                    ),
+                )
+                for idx, item in enumerate(self._src)
+            ),
+            tablefmt=tablefmt,
+            headers=(
+                "Idx",
```

Comment:
`idx` is very programmer-y. I'd use the full `"Index"`

Reply:
I was trying to lose characters, since the line was getting overly wide on text mode. How about "#"?